parm/metplus_config/prep/nwps/wave_grid2obs/PB2NC_wave.conf

```diff
@@ -0,0 +1,148 @@
+# PrepBufr to NetCDF Configurations
+
+# section heading for [config] variables - all items below this line and
+# before the next section heading correspond to the [config] section
+[config]
+
+# Redirect final configuration file
+# MET_INSTALL_DIR, MET_BIN_DIR, OUTPUT_BASE are defined in machine.conf
+
+INPUT_BASE = {ENV[COMINobsproc]}
+METPLUS_CONF = {ENV[DATA]}/metplus_final.conf
+
+# List of applications to run - only PB2NC for this case
+PROCESS_LIST = PB2NC
+
+# time looping - options are INIT, VALID, RETRO, and REALTIME
+# If set to INIT or RETRO:
+#   INIT_TIME_FMT, INIT_BEG, INIT_END, and INIT_INCREMENT must also be set
+# If set to VALID or REALTIME:
+#   VALID_TIME_FMT, VALID_BEG, VALID_END, and VALID_INCREMENT must also be set
+LOOP_BY = VALID
+
+# Format of VALID_BEG and VALID_END using % items
+# %Y = 4 digit year, %m = 2 digit month, %d = 2 digit day, etc.
+# %Y%m%d%H expands to YYYYMMDDHH
+VALID_TIME_FMT = %Y%m%d%H
+
+# Start time for METplus run - must match VALID_TIME_FMT
+VALID_BEG = {ENV[INITDATE]}{ENV[HH]}
+
+# End time for METplus run - must match VALID_TIME_FMT
+VALID_END = {ENV[INITDATE]}{ENV[HH]}
+
+# Increment between METplus runs (in seconds if no units are specified)
+#  Must be >= 60 seconds
+# these will be the output files times too
+VALID_INCREMENT = 6H
+
+# List of forecast leads to process for each run time (init or valid)
+# In hours if units are not specified
+# If unset, defaults to 0 (don't loop through forecast leads)
+LEAD_SEQ = 0
+INIT_SEQ = 
+
+# list of offsets in the prepBUFR input filenames to allow. List is in order of preference
+# i.e. if 12, 6 is listed, it will try to use a 12 offset file and then try to use a 6 offset
+# if the 12 does not exist
+PB2NC_OFFSETS = 0
+
+# Order of loops to process data - Options are times, processes
+# Not relevant if only one item is in the PROCESS_LIST
+# times = run all wrappers in the PROCESS_LIST for a single run time, then
+#   increment the run time and run all wrappers again until all times have
+#   been evaluated.
+# processes = run the first wrapper in the PROCESS_LIST for all times
+#   specified, then repeat for the next item in the PROCESS_LIST until all
+#   wrappers have been run
+LOOP_ORDER = times
+
+# Location of MET config file to pass to PB2NC
+PB2NC_CONFIG_FILE = {ENV[METPLUS_PATH]}/parm/met_config/PB2NCConfig_wrapped
+
+# If set to True, skip run if the output file determined by the output directory and
+# filename template already exists
+PB2NC_SKIP_IF_OUTPUT_EXISTS = False
+
+PB2NC_SKIP_TIMES = 
```

Comment:
`PB2NC_SKIP_TIMES` and `SKIP_TIMES` are no longer used in METplus. These can be removed.